src/main/avro/com.snowplowanalytics.sauna.observers/AmazonKinesisConfig/avro/1-0-0.avsc

```diff
@@ -0,0 +1,46 @@
+{
+  "namespace": "com.snowplowanalytics.sauna.observers",
+  "name": "AmazonKinesisConfig",
+  "type": "record",
+  "fields": [
+    {
+      "name": "enabled",
+      "type": "boolean"
+    },
+    {
+      "name": "id",
+      "type": "string"
+    },
+    { "name": "parameters",
+      "type": {
+        "name": "AmazonKinesisConfigParameters",
+        "type": "record",
+        "fields": [
+          {
+            "name": "awsRegion",
+            "type": "string",
+            "default": null
+          },
+          {
+            "name": "awsAccessKeyId",
+            "type": "string",
+            "default": null
```

Comment:
I agree that this should be done uniformly across anything that uses AWS Services.

Perhaps even abstracted out into its own management service. A class that checks for ENV variables, overridden if global configuration file exists, overridden by individual observer/responder configs if they exist.

In any case, it makes sense to force credentials to be provided here and then an AWS auth solution be a different PR.